app/test_main.py

```diff
@@ -1,3 +1,62 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        pytest.param(
+            14, 14, [0, 0],
+            id="should add zeros with animal ages less than 15"
+        ),
+        pytest.param(
+            23, 23, [1, 1],
+            id="add one when animal ages less than 24"
+        ),
+        pytest.param(
+            27, 28, [2, 2],
+            id="add one more year when dog is 28, cat is 27"
+        ),
+        pytest.param(
+            28, 29, [3, 3],
+            id="add one more year when dog is  28, cat is 27"
+        )
+    ]
```

Comment:
Add more test cases to cover all scenarios mentioned in the checklist, e.g. testing with negative numbers, large numbers and numbers that differ greatly between `cat_age` and `dog_age` to test that output does not change because of this input.